### PR TITLE
NEW: community distro wiring

### PR DESCRIPTION
**Pull request by lizgehret**

### CODE REVIEW

File: .github/workflows/cron-2022.4-core-ubuntu-latest.yaml [hunk 1]
```diff
@@ -10,8 +10,8 @@ env:
 on:
   workflow_dispatch: {}
   schedule:
-    # dev - every 2 hours
-    - cron: '37 */2 * * *'
+    # dev - every 4 hours, starting at the zeroth hour
```
Comment (thermokarst) [inline .github/workflows/cron-2022.4-core-ubuntu-latest.yaml:13]: I like this - every 2 hrs was feeling a bit much. Also, being able to run the jobs on-demand using `workflow_dispatch` is a nice escape hatch for time-sensitive situations.

**Comment by ebolyen** [comment]
Sir.

**Comment by ebolyen** [comment]
Well that's interesting.

**Comment by thermokarst** [comment]
How dare you, @q2d2

**Comment by ebolyen** [comment]
new PR here: #168

**Pull request merged**